apps/web/src/app/api/documentLogs/download-logs/route.ts

```diff
@@ -15,34 +26,36 @@ export const POST = errorHandler(
         workspace: Workspace
       },
     ) => {
-      const formData = await request.formData()
-      const idsRaw = formData.get('ids')
-
-      if (!idsRaw) {
-        throw new BadRequestError('No document log ids provided')
-      }
-
-      const ids: string[] = JSON.parse(idsRaw as string)
+      try {
+        const body = await request.json()
+        const {
+          documentUuid,
+          extendedFilterOptions,
+          staticColumnNames,
+          parameterColumnNames,
+        } = downloadLogsRequestSchema.parse(body)
```

Comment:
You have safeParse which gives `result.success`. So no need to try/catch for the zod errors